lib/archethic/bootstrap/transaction_handler.ex

```diff
@@ -92,6 +92,12 @@ defmodule Archethic.Bootstrap.TransactionHandler do
     origin_public_key_certificate = Crypto.get_key_certificate(origin_public_key)
 
     Transaction.new(:node, %TransactionData{
+      code: """
+        condition inherit: [
+          # We need to ensure the type stays consistent
+          type: :node
+        ]
```

Comment:
I think you have to allow the `content` change, as node can change ip or port or reward address

Reply:
Yes, that's better.